tests/integration_tests/test_hyperopt.py

```diff
@@ -158,9 +158,13 @@ def test_hyperopt_run_hyperopt(csv_filename, samplers):
             output_feature_name
             + ".fc_layers": {
                 "type": "category",
-                "values": [[{"fc_size": 512}, {"fc_size": 256}], [{"fc_size": 512}], [{"fc_size": 256}]],
+                "values": [
+                    [{"output_size": 512}, {"output_size": 256}],
+                    [{"output_size": 512}],
+                    [{"output_size": 256}],
+                ],
```

Comment:
these values are likely too big, let's decrease it for faster tests

Reply:
Done